### PR TITLE
website: made changes to landing page wrt pods and screenshots

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### What does this PR do?

Added section for pods as well as updated screenshot on the Hero section of the landing page

### Screenshot/screencast of this PR
Screen Shot 2022-10-24 at 9 57 30 AM
Screen Shot 2022-10-24 at 9 57 36 AM

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
 
+function Pods() {
+  return (
+    <section className="text-gray-600 dark:text-gray-300 dark:bg-zinc-800 bg-zinc-200 body-font py-24">
+      <div className="container px-5 mx-auto flex flex-wrap">
+        <div className="flex flex-col text-center w-full mb-5">
+          <SectionTitle name="features" />
+
+          <h2 className="max-w-lg mb-6 font-sans text-3xl font-bold leading-none tracking-tight text-gray-900 dark:text-white sm:text-4xl md:mx-auto">
+            Work with Pods.
```
Comment (cdrage) [inline website/src/pages/index.tsx]: I feel like the period shouldn't be here?

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+            </div>
+            <div className="p-4 md:w-1/2 flex">
+              <div className="w-12 h-12 inline-flex items-center justify-center rounded-full bg-gray-400 text-purple-900 mb-4 flex-shrink-0">
+                <FontAwesomeIcon size="2x" icon={faGaugeHigh} className="w-6 h-6 " />
+              </div>
+              <div className="flex-grow pl-6">
+                <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Kubernetes</h2>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Play Kubernetes YAML locally without Kubernetes
```
Comment (cdrage) [inline website/src/pages/index.tsx]: Doesn't make sense. Play Kubernetes without Kubernetes. You mean maybe: `locally without kubectl`?

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Generate Kubernetes YAML from Pods
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Leverage existing remote Kubernetes cluster
```
Comment (cdrage) [inline website/src/pages/index.tsx]:
```suggestion
                  Leverage existing remote Kubernetes clusters
```

**Review by cdrage — CHANGES_REQUESTED** [review]

File: website/src/pages/index.tsx (hunk 2 above)
Comment (deekay2310) [reply website/src/pages/index.tsx]: I've used the words from the slides @slemeur sent me over

**Comment by benoitf** [comment]
FYI I extracted the logo change of this PR and merged it
https://github.com/containers/podman-desktop/pull/687

**Comment by deekay2310** [comment]
Yes, I came across that. Thank you for the PR @benoitf

**Comment by deekay2310** [comment]
Screen Shot 2022-10-25 at 8 55 38 AM

File: website/src/pages/index.tsx [hunk 4]
```diff
@@ -365,7 +365,7 @@ function EnterpriseReady() {
                 <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Code signing</h2>
 
                 <p className="leading-relaxed text-base">
-                  macOS binaries are digitally signed (Windows certification is in)
+                  Digitally signed binaries available across all leading Operating Systems
```
Comment (benoitf) [inline website/src/pages/index.tsx]:
```suggestion
                  Digitally signed binaries available on Windows and macOS
```

Comment (deekay2310) [reply website/src/pages/index.tsx]: Yeah, thought that could be the case. Thanks for the tweak.

File: website/tailwind.config.js [hunk 5]
```diff
@@ -6,7 +6,7 @@ module.exports = {
   theme: {
     extend: {
       backgroundImage: {
-        'hero-pattern': "url('/img/gradients.png')",
+        'hero-pattern': 'url(\'/img/gradients.png\')',
```
Comment (benoitf) [inline website/tailwind.config.js]: you need to remove that change for now for the linter check

**Review by benoitf — COMMENTED** [review]
you'll need to remove the png from this PR

**Comment by deekay2310** [comment]
The one on the homepage? I just merged the new changes. It should be fine, right?

File: website/tailwind.config.js (hunk 5 above)
Comment (deekay2310) [reply website/tailwind.config.js]: done

**Comment by benoitf** [comment]
> The one on the homepage? I just merged the new changes. It should be fine, right?

no, it's still trying to add the file

File: website/src/pages/index.tsx (hunk 2 above)
Comment (benoitf) [reply website/src/pages/index.tsx]: @cdrage you only need Podman engine, not kubernetes engine

but maybe rephrase then to:
``` Play Kubernetes YAML directly with Podman Engine```

Comment (deekay2310) [reply website/src/pages/index.tsx]: done!
https://github.com/containers/podman-desktop/pull/683/commits/1079ff1791982f18707b118a64b8ab4582913cd1

File: website/src/pages/index.tsx [hunk 6]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
 
+function Pods() {
+  return (
+    <section className="text-gray-600 dark:text-gray-300 dark:bg-zinc-800 bg-zinc-200 body-font py-24">
+      <div className="container px-5 mx-auto flex flex-wrap">
+        <div className="flex flex-col text-center w-full mb-5">
+          <SectionTitle name="features" />
+
+          <h2 className="max-w-lg mb-6 font-sans text-3xl font-bold leading-none tracking-tight text-gray-900 dark:text-white sm:text-4xl md:mx-auto">
+            Work with Pods
```
Comment (slemeur) [inline website/src/pages/index.tsx]: Work with Pods and Kubernetes

File: website/src/pages/index.tsx [hunk 7]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Play Kubernetes YAML directly with Podman Engine
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Generate Kubernetes YAML from Pods
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Leverage existing remote Kubernetes clusters
```
Comment (slemeur) [inline website/src/pages/index.tsx]: Deploy to existing Kubernetes environments

File: website/src/pages/index.tsx [hunk 8]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+                <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">
+                  Create & Select
+                </h2>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faGaugeHigh} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Create and Start Pods with Podman Desktop
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faDiagramProject} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Select Containers to run as Pods
```
Comment (slemeur) [inline website/src/pages/index.tsx]: Create Pods from existing Containers

File: website/src/pages/index.tsx [hunk 9]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+              <div className="w-12 h-12 inline-flex items-center justify-center rounded-full bg-gray-400 text-purple-800 mb-4 flex-shrink-0">
+                <FontAwesomeIcon size="2x" icon={faGears} className="w-6 h-6 " />
+              </div>
+              <div className="flex-grow pl-6">
+                <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">
+                  Create & Select
+                </h2>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faGaugeHigh} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Create and Start Pods with Podman Desktop
```
Comment (slemeur) [inline website/src/pages/index.tsx]: Create, Start and Manage Pods

File: website/src/pages/index.tsx (hunk 6 above)
Comment (deekay2310) [reply website/src/pages/index.tsx]: https://github.com/containers/podman-desktop/pull/683/commits/5b496a6674e737c12ecb6eb30e5b28ee7c69b27f

File: website/src/pages/index.tsx (hunk 7 above)
Comment (deekay2310) [reply website/src/pages/index.tsx]: https://github.com/containers/podman-desktop/pull/683/commits/5b496a6674e737c12ecb6eb30e5b28ee7c69b27f

File: website/src/pages/index.tsx (hunk 8 above)
Comment (deekay2310) [reply website/src/pages/index.tsx]: https://github.com/containers/podman-desktop/pull/683/commits/5b496a6674e737c12ecb6eb30e5b28ee7c69b27f

File: website/src/pages/index.tsx (hunk 9 above)
Comment (deekay2310) [reply website/src/pages/index.tsx]: https://github.com/containers/podman-desktop/pull/683/commits/5b496a6674e737c12ecb6eb30e5b28ee7c69b27f

**Review by slemeur — APPROVED** [review]
LGTM

**Comment by deekay2310** [comment]
@slemeur if you could sign your commit, i'll be able to merge the PR

File: website/src/pages/index.tsx [hunk 10]
```diff
@@ -365,7 +365,7 @@ function EnterpriseReady() {
                 <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Code signing</h2>
 
                 <p className="leading-relaxed text-base">
-                  macOS binaries are digitally signed (Windows certification is in)
+                  Digitally signed binaries available across Windows and macOS.
```
Comment (cdrage) [inline website/src/pages/index.tsx]: Remove the period at the end :)

File: website/src/pages/index.tsx [hunk 11]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+        </div>
+        <div className="container px-5 pb-5 mx-auto">
+          <div className="flex flex-wrap sm:-m-4 -mx-4 -mb-10 -mt-4 md:space-y-0 space-y-6">
+            <div className="p-4 md:w-1/2 flex">
+              <div className="w-12 h-12 inline-flex items-center justify-center rounded-full bg-gray-400 text-purple-800 mb-4 flex-shrink-0">
+                <FontAwesomeIcon size="2x" icon={faGears} className="w-6 h-6 " />
+              </div>
+              <div className="flex-grow pl-6">
+                <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">
+                  Create & Pod-ify
```
Comment (cdrage) [inline website/src/pages/index.tsx]:
```suggestion
                  Create and "Podify"
```

Pod-ify is incorrect grammar as joining words have to both be words (ify is not a word).

"Podify" with quotes may be better.

File: website/src/pages/index.tsx [hunk 12]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+                </h2>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faGaugeHigh} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Create, Start and Manage Pods with Podman Desktop
```
Comment (cdrage) [inline website/src/pages/index.tsx]:
```suggestion
                  Create, start, and manage pods with Podman Desktop
```

Incorrect titlelization and oxford comma

File: website/src/pages/index.tsx [hunk 13]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faDiagramProject} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Create Pods from existing Containers
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Start / Stop / Restart Pods
```
Comment (cdrage) [inline website/src/pages/index.tsx]: This is repeated again :( you already say 'create, start and manage" pods, and this part is start / stop / restart. May make sense for something else?

File: website/src/pages/index.tsx [hunk 14]
```diff
@@ -591,6 +587,90 @@ function MainFeatures() {
...
+              </div>
+              <div className="flex-grow pl-6">
+                <h2 className="text-gray-900 dark:text-gray-100 text-lg title-font font-medium mb-2">Kubernetes</h2>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Play Kubernetes YAML directly with Podman Engine
+                </p>
+                <p className="leading-relaxed text-base list-disc">
+                  <FontAwesomeIcon icon={faRocket} className="text-purple-800 w-3 h-3 mt-1 mr-2" />
+                  Generate Kubernetes YAML from Pods
```
Comment (cdrage) [inline website/src/pages/index.tsx]:
```suggestion
                  Generate Kubernetes YAML from pods
```

File: website/src/pages/index.tsx (hunk 10 above)
Comment (cdrage) [reply website/src/pages/index.tsx]: Available kinda sounds odd / not confident? Maybe something like this would be better.

Comment (cdrage) [inline website/src/pages/index.tsx]:
```suggestion
                  Signed binaries for both macOS and Windows
```

And maybe prefer macOS first because usually dev's are: #1 Mac or Linux and then Windows

**Review by cdrage — CHANGES_REQUESTED** [review]

**Review by slemeur — APPROVED** [review]
LGTM

**Comment by deekay2310** [comment]
@slemeur need you to sign your commits - 
1. https://github.com/containers/podman-desktop/pull/683/commits/5c963a854bd3ddb4fd9227cc1cf5abe60fec4966
2. https://github.com/containers/podman-desktop/pull/683/commits/83e9aec9756949930955a006fafa62978aa28e61

**Comment by deekay2310** [comment]
@cdrage need you to approve the changes in order to merge the PR

**Review by cdrage — APPROVED** [review]
